File: Frontend/src/contracts/responses/airports/airportSearchResponse.ts
// TypeScript-тип airportSearchResponse описывает форму данных, которые frontend получает от backend.

export interface AirportSearchResponse {
    id: number;
    name: string;
    city?: string;
    country: string;
    iata?: string | null;
    icao?: string | null;
    latitude: number;
    longitude: number;
}

File: AirCharter.API/AirCharter.API/Responses/Airports/AirportSearchResponse.cs
namespace AirCharter.API.Responses.Airports;

public sealed class AirportSearchResponse
{
    public int Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public string? City { get; set; }

    public string Country { get; set; } = string.Empty;

    public string? Iata { get; set; }

    public string? Icao { get; set; }

    public decimal Latitude { get; set; }

    public decimal Longitude { get; set; }
}